manual/Searching/Full_text_matching/Basic_usage.md

```diff
@@ -118,6 +118,17 @@ By default, keywords are combined using the OR operator. However, you can change
 }
 ```
 
+### match_all
+
+"match_all" accepts an empty object and returns documents from the index without any text matching. It is similar to the same query used in Elasticsearch. Alternatively, you can just omit the `query` clause in the request which will have the same effect. 
```

Comment:
- index
- table

Reply:
"without any text matching" sounds ambiguous.

Reply:
"It is similar to the same query used in Elasticsearch. " is unnecessary. Pls remove it.